src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/MemoryFormStore.js

```diff
@@ -41,6 +43,8 @@ export default class MemoryFormStore extends AbstractFormStore implements FormSt
 
     @action setMultiple(data: Object) {
         this.data = data;
+
+        super.setMultiple();
```

Comment:
Because currently the parent class does not have any use for the data. And introducing it would make eslint fail, which would mean we would have to ignore that error. Instead I prefer to not pass that information, since the abstract super class can't set the data anyway.